apps/web/screens/OfficialJournalOfIceland/OJOISubpage.tsx

```diff
@@ -0,0 +1,120 @@
+import { useIntl } from 'react-intl'
+
+import { SliceType } from '@island.is/island-ui/contentful'
+import { Box } from '@island.is/island-ui/core'
+import { Locale } from '@island.is/shared/types'
+import {
+  ContentLanguage,
+  CustomPage,
+  CustomPageUniqueIdentifier,
+  Query,
+  QueryGetCustomSubpageArgs,
+  QueryGetOrganizationArgs,
+} from '@island.is/web/graphql/schema'
+import { useLinkResolver } from '@island.is/web/hooks'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import { CustomNextError } from '@island.is/web/units/errors'
+import { webRichText } from '@island.is/web/utils/richText'
+
+import { OJOIWrapper } from '../../components/OfficialJournalOfIceland'
+import { CustomScreen, withCustomPageWrapper } from '../CustomPage'
+import { GET_ORGANIZATION_QUERY } from '../queries'
+import { GET_CUSTOM_SUBPAGE_QUERY } from '../queries/CustomPage'
+import { ORGANIZATION_SLUG } from './constants'
+import { m } from './messages'
+
+interface OJOISubpageProps {
+  organization?: Query['getOrganization']
+  locale: Locale
+  customSubpage?: CustomPage
+}
+
+const OJOISubpage: CustomScreen<OJOISubpageProps> = ({
+  organization,
+  locale,
+  customSubpage,
+}) => {
+  const { formatMessage } = useIntl()
+  const { linkResolver } = useLinkResolver()
+
+  return (
+    <OJOIWrapper
+      pageTitle={customSubpage?.ogTitle ?? ''}
+      pageDescription={customSubpage?.ogDescription ?? ''}
+      organization={organization ?? undefined}
+      pageFeaturedImage={customSubpage?.ogImage?.url}
+      breadcrumbItems={[
+        {
+          title: formatMessage(m.breadcrumb.frontpage),
+          href: linkResolver('homepage', [], locale).href,
+        },
+        {
+          title: formatMessage(m.home.title),
+          href: linkResolver('ojoihome', [], locale).href,
```

Comment:
Fun fact, you actually don't gave to provide the locale here since you are using the `useLinkResolver` hook.